go.mod

```diff
@@ -1,65 +1,19 @@
 module gopkg.in/src-d/go-mysql-server.v0
 
 require (
```

Comment:
Is it normal that all that dependencies disappear?

Reply:
I'm not sure how vgo builds this dependency graph, but as you see everything what disappeared are indirect deps. The good thing is that it still builds.